FakeItEasy.ruleset

```diff
@@ -5,4 +5,17 @@
     <Rule Id="CA1020" Action="None" />
     <Rule Id="CA1305" Action="None" />
   </Rules>
+  <Rules AnalyzerId="StyleCop.Analyzer" RuleNamespace="StyleCop.Analyzer">
+    <Rule Id="SA1117" Action="None" />

```

Comment:
In general, I'm loathe to turn off the rules we previously had on. I'm not sure why these weren't caught before.
There are only about 8.

Me, I would be content with a "clean up the rules" issue to be performed later.